File: wheere-renewal-front/wheere_driver/lib/model/dto/station_dto.dart
class StationDTO{
  int sId;
  int sSeq;
  String sName;

  StationDTO({
    required this.sId,
    required this.sSeq,
    required this.sName
  });

  factory StationDTO.fromJson(Map<String, dynamic> json) {
    return StationDTO(
      sId: json['sId'],
      sSeq: json['sSeq'],
      sName: json['name'],
    );
  }
}

File: wheere-renewal-back/wheere/src/main/java/kr/ac/kumoh/sjjpl138/wheere/platform/dto/StationDto.java
package kr.ac.kumoh.sjjpl138.wheere.platform.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import kr.ac.kumoh.sjjpl138.wheere.platform.Platform;
import lombok.Data;

@Data
public class StationDto {
        @JsonProperty("sId")
        private Long sId;
        @JsonProperty("sName")
        private String sName;
        @JsonProperty("sSeq")
        private int sSeq;

        public StationDto(Platform platform) {
                this.sId = platform.getStation().getId();
                this.sName = platform.getStation().getName();
                this.sSeq = platform.getStationSeq();
        }
}
